src/content/guides/scaffolding.md

```diff
@@ -7,17 +7,21 @@ contributors:
   - EugeneHlushko
 ---
 
-It can be hard to set up a complex webpack configuration for the first time. Writing advanced configurations to optimize performance is quite hard. The `init` feature is designed to support people that want to create their own configuration or initializing projects that other people create.
+It can be hard to set up a complex webpack configuration for the first time. Writing advanced configurations to optimize performance could be even more difficult. The `init` feature is designed to support people that want to create their own configuration or initializing projects that other people create.
 
 ## Creating a scaffold
 
-Before writing a `webpack-cli` scaffold, think about what you're trying to achieve. Do you want a "general" scaffold that could be used by any project or type of app? Do you want something focused, like a scaffold that writes both your `webpack.config.js` and your framework code? It's also useful to think about the user experience for your scaffold.
+Before writing a `webpack-cli` scaffold, think about what you're trying to achieve and who is going to use it:
+
+- Do you want a generalized scaffold that could be used by any project or type of application?
+- Do you want something specific, like a scaffold that writes both your `webpack.config.js` and your framework code?
+- Who are the potential users and what user experience will look like for the users of your scaffold?
 
 `webpack-cli` offers an interactive experience to customize the output accordingly. For example asking questions like: "What is your entry point?".
 
 ### Writing a scaffold
 
-There are many resources where you can learn how to write a scaffold, you can start from [Writing a Scaffold](/contribute/writing-a-scaffold/)
+There are many resources where you can learn how to write a scaffold, you can start from [Writing a Scaffold page](/contribute/writing-a-scaffold/).
```

Comment:
```suggestion
There are many resources where you can learn how to write a scaffold, you can start from [Writing a Scaffold](/contribute/writing-a-scaffold/) guide.
```

Reply:
This is the writing a scaffold guide though, so it cannot link to another page saying that guide is there :(

Reply:
on the other hand it says `scaffolding` guide, and the other page is `writing a scaffold` so maybe:
```suggestion
There are many resources where you can learn how to write a scaffold, you can start from reading about [Writing a Scaffold](/contribute/writing-a-scaffold/).
```

Reply:
Yup, This seems right (Maybe `tutorial` is the word we are looking for?).
Also `There are many resources where you can learn how to write a scaffold` is a bit of an overstatement in my opinion, maybe we can have a better start for this sentence.

Reply:
Yes, potentially
```suggestion
There are various resources where you can learn how to write a scaffold, you can start by reading [Writing a Scaffold](/contribute/writing-a-scaffold/) tutorial.
```

Reply:
Let's go ahead with this one